app/controllers/user_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UserController < ApplicationController
```

Comment:
- [ ] By convention controller names are often in plural form, not singular so this file name should be `users_controller.rb`. Kindly rename this file.

- [ ] The class should also be `UsersController`.